Doc/library/sqlite3.rst

```diff
@@ -98,6 +71,32 @@ This example uses the iterator form::
    ('2006-04-05', 'BUY', 'MSFT', 1000, 72.0)
 
 
+.. _sqlite3-placeholders:
+
+Usually your SQL operations will need to use values from Python variables.  You
+shouldn't assemble your query using Python's string operations because doing so
+is insecure; it makes your program vulnerable to an SQL injection attack
+(see https://xkcd.com/327/ for a humorous example of what can go wrong)::
+
+   # Never do this -- insecure!
+   symbol = 'RHAT'
+   cur.execute("SELECT * FROM stocks WHERE symbol = '%s'" % symbol)
+
+Instead, use the DB-API's parameter substitution. Put a placeholder wherever
```

Comment:
@berkerpeksag I'd like to rewrite this section further. Can I reuse the issue number for a second PR? Plan:

- Don't start with the how-to-do-it-wrong; start with best practise, and make it more concise.
- Move SQL injection attack example to "Security Considerations" at the end of the docs.

Reply:
> * Don't start with the how-to-do-it-wrong; start with best practise, and make it more concise.

That would require rewriting pretty much everything. I prefer keeping it as is.